packages/backend-core/src/objectStore/utils.ts

```diff
@@ -71,7 +72,7 @@ async function processUrlAttachment(
   const fallbackFilename = path.basename(new URL(attachment.url).pathname)
   return {
     filename: attachment.filename || fallbackFilename,
-    content: response.body,
+    content: stream.Readable.fromWeb(response.body as streamWeb.ReadableStream),
```

Comment:
Will this always be available?

Reply:
Good point - in theory there always should be a response body, but it might not always be a stream.